toptal/picasso
# feat(OverlayLoader): [FX-1174] add OverlayLoader component

[FX-1174]

### Description

Add a OverlayLoader component to Picasso-lab to be used when rendered data is refetched, i.e. during pagination or filter changes. The component renders a semi-transparent layer on top of child UI and an animated spinner.

### How to test

- Start Storybook `yarn start`
- Navigate to `Lab/OverlayLoader` 
- Make sure it renders correctly

### Screenshots

![image](https://user-images.githubusercontent.com/254110/100613751-b5143280-331d-11eb-80b2-5fef3f3220f1.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-1174]: https://toptal-core.atlassian.net/browse/FX-1174
